src/index.ts

```diff
@@ -64,8 +64,15 @@ export const withPgAdapter = <PrismaClientType>(
     },
   })
 
-  const { url } = prisma._engineConfig.overrideDatasources?.db ?? prisma._engineConfig.inlineDatasources.db
-  const pool = new Pool({ connectionString: url.value || process.env[url.fromEnvVar] });
+  let connectionString;
+  if (prisma._engineConfig.overrideDatasources?.db?.url) {
+    connectionString = prisma._engineConfig.db.url;
```

Comment:
Hey @mattmediavine — should this be `prisma._engineConfig.overrideDatasources?.db?.url`?

Reply:
Nope, fat fingered. I noticed this also and already made a change

Reply:
It should be one optional as overrideDatasources may not have a db property if it is not being used.

Reply:
Sorry, to clarify, it seems like `*.overrideDatasources.*` is missing in this chain?

Reply:
you were absolutely correct. Sorry I have fixed that issue.

Reply:
Thank you! 🙌 

This should be released in `v0.8.1`